addons/grenadethrowing/functions/fnc_initControls.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Dslyecxi, Jonpas
+ * Initializes or deinitializes controls.
+ *
+ * Arguments:
+ * 0: Initialize/Deinitialize Controls <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [true] call ace_grenadethrowing_fnc_initControls
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_enable"];
+
+if (_enable) then {
+    GVAR(KeyDownHandle) = (findDisplay 46) displayAddEventHandler ["KeyDown", {_this call FUNC(onKeyDown)}];
+    GVAR(MouseButtonDownHandle) = (findDisplay 46) displayAddEventHandler ["MouseButtonDown", {_this call FUNC(onMouseButtonDown)}];
+    GVAR(MouseScrollHandle) = (findDisplay 46) displayAddEventHandler ["MouseZChanged", {_this call FUNC(onMouseScroll)}];
```

Comment:
Will wait on fixing that when key events are fixed when used with this on CBA's side.